samples/simple-bounce.rb

```diff
@@ -1,24 +1,31 @@
-xspeed, yspeed = 8.4, 6.6
+#The Shoes logo/icon bounces on the screen. The logo/icon responds to menu options and if clicked on.
+xspeed, yspeed = 10, 6
 xdir, ydir = 1, 1
 
-Shoes.app do
-  background "#DFA"
-  border black, strokewidth: 6
-
-  nostroke
-  @icon = image "#{Shoes::DIR}/static/shoes-icon.png", left: 100, top: 100 do
-    alert "You're soooo quick."
+Shoes.app width: 300, height: 300 do
+  a = nil
+  button('toggle') {a.toggle}
+  button('stop') {a.stop}
+  button('start') {a.start}
+  button('remove') {a.remove}
+  img = image File.join(Shoes::DIR, 'static/shoes-icon.png') do
+    alert "You're soooo quick!"
   end
 
-  x, y = self.width / 2, self.height / 2
-  size = [@icon.height, @icon.width]
-  animate(30) do
-    x += xspeed * xdir
-    y += yspeed * ydir
+  x, y = 150, 150
+  size = [128, 128]
+  pause = 0
+
+  a = animate 24 do |n|
+    unless pause == n
+      x += xspeed * xdir
+      y += yspeed * ydir
 
-    xdir *= -1 if x > self.width - size[0] or x < 0
-    ydir *= -1 if y > self.height - size[1] or y < 0
+      xdir *= -1 if x > 300 - size[0] or x < 0
+      ydir *= -1 if y > 300 - size[1] or y < 0
 
-    @icon.move x.to_i, y.to_i
+      img.move x.to_i, y.to_i
+    end
+    pause = n
   end
 end
```

Comment:
Seems to me that you accidentally overrode somple-bounce.rb with some other sample?